transaction/src/action.rs

```diff
@@ -169,4 +172,6 @@ impl TryFrom<pb::Action> for Action {
             }
         }
     }
+
+    type Error = anyhow::Error;
```

Comment:
probably no particular reason to move this out of the usual place?

Reply:
I thought so too, however:

```error[E0046]: not all trait items implemented, missing: `Error`
   --> transaction/src/action.rs:136:1
    |
136 | impl TryFrom<pb::Action> for Action {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ missing `Error` in implementation
    |
    = help: implement the missing item: `type Error = Type;`

For more information about this error, try `rustc --explain E0046`.
error: could not compile `penumbra-transaction` due to previous error```